### DISCUSSION TITLE
I'd love to give this a go, but....

### DISCUSSION THREAD

**Discussion by makeclean** [leap71/PicoGK#23] [General]
..... I'm a Linux user. Are there any plans to supply the C++ source so I could give it a go?

**Comment by LinKayser** [comment]
Hi Andrew,

the source code for the PicoGK Runtime is also open source and available here: https://github.com/leap71/PicoGKRuntime

@CorrieVS ported the last version to LINUX, so maybe check her fork: https://github.com/CorrieVS/PicoGKRuntime 

Let me know how this goes. It should be relatively straightforward.

Best,
Lin

**Reply by makeclean** [reply]
Yeah, I got it compiled on Linux last night, I needed to make some changes for gcc with respect to some header files - very minor changes related to structs with complex constructors. It wasn't clear for me however, how to use the PikoGS C# files with the compiled libraries? Or should I be just using the C++ interface directly?

**Reply by LinKayser** [reply]
If you change the path as described [here](https://github.com/leap71/PicoGK/blob/main/Documentation/Compiling_PicoGKRuntime.md#using-the-compiled-runtime) you can run the C# side with your custom compiled dylib. A lot of functionality is on the C# side, so just using the C++ side would be extremely limiting.

Best,
Lin
